### PR TITLE
first draft of NOTE re: `usePolling`

### PR DESCRIPTION
**Pull request by michaelsbradleyjr**
Pertains to #297. Please consider it a first-draft; I'll be happy to make revisions in a timely fashion.

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -680,6 +680,8 @@ function(nwb) {
 
 nwb's default [Karma](http://karma-runner.github.io/) configuration uses the [Mocha](https://mochajs.org/) framework and reporter plugins for it, but you can configure your own preferences.
 
+**NOTE:**&nbsp; At runtime, Karma sets a little documented `usePolling` autoWatch option to `true` [if the platform is detected to be macOS or Linux](https://github.com/karma-runner/karma/blob/master/lib/config.js#L318). However, Karma's non-polling file-watching works correctly and consumes dramatically less CPU on macOS, at least as recently as Sierra (10.12). Users on macOS will want to set `usePolling: false` within the [`extra:`](#extra-object-1) Object in the `karma:` config section of their `nwb.config.js`.
```
Comment (loklaan) [inline docs/Configuration.md]: Reviewed this yesterday - didn't submit the review until today, sorry.

Comment (michaelsbradleyjr) [inline docs/Configuration.md]: Okay, will revise and push another commit later today.

**Comment by michaelsbradleyjr** [comment]
Revisions submitted.

**Comment by loklaan** [comment]
Nice one, thanks for doing that!

FYI I'll be squashing the commits.

**Comment by loklaan** [comment]
Merge in 7805932

**Comment by loklaan** [comment]
Resolves #298.

**Comment by loklaan** [comment]
Resolves #297

**Pull request closed without merging**